orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccountInterventionService.java

```diff
@@ -17,26 +17,20 @@
 public class AccountInterventionService {
 
     private static final Logger LOGGER = LogManager.getLogger(AccountInterventionService.class);
-    private final boolean accountInterventionsEnabled;
     private final HttpClient httpClient;
     private final URI accountInterventionServiceURI;
 
     public AccountInterventionService(ConfigurationService configService, HttpClient httpClient) {
-        this.accountInterventionsEnabled = configService.isAccountInterventionServiceEnabled();
         this.accountInterventionServiceURI = configService.getAccountInterventionServiceURI();
         this.httpClient = httpClient;
     }
 
     public AccountInterventionStatus getAccountStatus(String internalPairwiseSubjectId)
             throws AccountInterventionException {
         try {
-            if (accountInterventionsEnabled) {
-                return retrieveAccountStatus(internalPairwiseSubjectId);
-            }
-
-            return new AccountInterventionStatus(false, false, false, false);
-
+            return retrieveAccountStatus(internalPairwiseSubjectId);
         } catch (IOException | URISyntaxException | InterruptedException | Json.JsonException e) {
+            Thread.currentThread().interrupt();
```

Comment:
We should only be setting this when catching an interrupted exception, not any of the others